voluncheer/.ebextensions/03_archive_cron.config

```diff
@@ -8,6 +8,7 @@ files:
             source /var/app/venv/*/bin/activate
             cd /var/app/current
             export $(cat /opt/elasticbeanstalk/deployment/env | xargs)
+            export $(cat /opt/elasticbeanstalk/deployment/env | xargs)
```

Comment:
This is here twice, seems like an oversight.

Reply:
Good catch!